docs/operate/get-started/other-hardware/micro-module.md

```diff
@@ -225,6 +225,54 @@ To create a new module compatible with the Micro-RDK, follow these steps.
 
 For further details on Micro-RDK development, including credentials management and developer productivity suggestions, please see the [development technical notes page on GitHub](https://github.com/viamrobotics/micro-rdk/blob/main/DEVELOPMENT.md).
 
+## Over-the-air updates
+
+To remotely update the firmware on your microcontroller without a physical connection to the device, add the OTA (over-the-air) service to your microcontroller's configuration in the [Viam app](https://app.viam.com).
+Use **JSON** mode to add the service as in the template below, then configure the URL from which to fetch new firmware, and the version name.
+
+When the `version` field is modified, the Micro-RDK will automatically download and install the firmware at the URL onto your device.
```

Comment:
According to the scope, I think this would be more accurate as:

```
Your device checks for configuration updates periodically. If the devices receives a configuration with the OTA service and a modified  version in it, the device downloads the new firmware to an inactive partition and restarts. When the device boots it will load the new firmware.
```

The main point being that the Micro-RDK does not download anything. The Micro-RDK isn't an executable, it's what the device firmwares are built from. This needs SME review though.

Reply:
@npentrel yes that correction is more accurate; however I think in your comment you may have meant 'does not install anything' instead of 'does not download anything'. 

It's an all-or-nothing process since it's similar to trying to swapping engines while driving. At least currently, there is no direct way to roll-back to a previous firmware if you do a bad upgrade. It's very possible to update to broken firmware that can leave the device in a bad state where it needs to be reflashed entirely